packages/manager/src/components/DatePicker/DateTimePicker.tsx

```diff
@@ -228,6 +234,12 @@ export const DateTimePicker = ({
             {showTime && (
               <Grid item xs={4}>
                 <TimePicker
+                  minTime={
+                    minDate &&
+                    minDate.toISODate() === selectedDateTime?.toISODate()
```

Comment:
```suggestion
                    minDate?.toISODate() === selectedDateTime?.toISODate()
```

Can we get rid of two conditions like this